Sources/XCHammer/XCBuildSettings.swift

```diff
@@ -129,9 +129,12 @@ struct Setting<T: XCSettingStringEncodeable & Semigroup>: Semigroup {
     }
 }
 
+
+
+
 struct XCBuildSettings: Encodable {
     var copts: [String] = []
-    var productName: First<String>?
+    var productName: First<String>? = First("$(TARGET_NAME)")
```

Comment:
So if we allocate this, it sets it to target name as the default? If we want to always set it to `TARGET_NAME` we should just remove the variable if it isn't necessary